docs/api-aggregation-setup.md

```diff
@@ -0,0 +1,183 @@
+
+# Setting up Service Catalog for API Aggregation in Kubernetes
+
+The aggregator is a server that sits in front of everything. 
```

Comment:
Might be nice to mention that what you mean is the aggregator sits in front of the apiserver